HLeithner/joomla-cms
# Add prepared statements for plg_actionlog_joomla

### Summary of Changes
Updated SQL queries to prepared statements and made some cleanups around the queries.


### Testing Instructions
Use the plugin in all ways you can think of.

* Activate the privacy extension with actionlog
* change the state of a article
* You should see the change in the actionlog as without the PR


### Expected result
Nothing changed.
